source/common/router/rds_impl.cc

```diff
@@ -174,5 +184,44 @@ Router::RouteConfigProviderSharedPtr RouteConfigProviderManagerImpl::getRouteCon
   return new_provider;
 };
 
+void RouteConfigProviderManagerImpl::addRouteInfo(RouteConfigProviderSharedPtr provider,
+                                                  Buffer::Instance& response) {
+  response.add(fmt::format(
+      "route_config_name: {}\n",
+      std::static_pointer_cast<RdsRouteConfigProviderImpl>(provider)->route_config_name_));
```

Comment:
Note, one way of doing this is to have RdsRouteConfigProvider interface, and only expose this for iteration. Another way is to also allow printing static route tables and differentiate them somehow.